pocketoptionapi_async/client.py

```
@@ -364,146 +418,168 @@ async def place_order(self, asset: str, amount: float, direction: OrderDirection
             # Wait for result (this will either get the real server response or create a fallback)
             result = await self._wait_for_order_result(order_id, order)
-            
+
             # Don't store again - _wait_for_order_result already handles storage
             logger.info(f"Order placed: {result.order_id} - {result.status}")
             return result
-            
+
         except Exception as e:
             logger.error(f"Order placement failed: {e}")
             raise OrderError(f"Failed to place order: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)